assets/component-image-with-text.css

```diff
@@ -299,6 +297,14 @@
   z-index: 2;
 }
 
+.image-with-text:not(.image-with-text--overlap) .color-background-1 {
+  background: transparent;
+}
+
+.image-with-text .gradient:not(.color-background-1) {
+  background-attachment: local;
+}
```

Comment:
I think this should also be applied when it's `.color-background-1` and set to Overlap. Also In that same situation, setting the image background to transparent maybe ? Checkout the border and background applied to the image with a transparent background:
![](https://screenshot.click/18-59-zz8kc-te21d.png)

Reply:
cc: @kmeleta

Reply:
Right, so in light of our decision to minimize (or completely avoid) background-1 type exceptions, I'm still opting to treat at least the overlap with local gradient in all cases for now. Additionally I feel making the content transparent can end up wonky pretty easily and making the media transparent without the content feels odd to me. But entirely possible still if we want to push back on that more.

Reply:
I think so, yes. Especially for this overlap version. 

In general though, I'm starting to get the feeling that we should make a few more exceptions for `background-1`. [Use cases like this](https://github.com/Shopify/dawn/pull/2651#discussion_r1202866791), as well as [maybe product cards](https://github.com/Shopify/dawn/pull/2651#discussion_r1202857830) feel like they could benefit from that. 

I know that breaks our consistency decision a little bit, but seeing this in action, I think we'll have slightly better visual outcomes if we make a few more exceptions. I'll do some more testing in the morning and form a more cohesive opinion.

Reply:
Just closing the loop here: I think we're fine with the cards approach as-is. If merchants want to have the backgrounds be transparent, they can change from "Card" to "Standard" in most cases.